config/initializers/urls.rb

```diff
@@ -1,23 +1,23 @@
-require './lib/google_api/drive.rb'
+# require './lib/google_api/drive.rb'
```

Comment:
Why is this all commented out?  I don't think you meant to push this, but your changes to this file (and `students.rb`) should not be part of this pull request and should not be merged to master.